### ISSUE TITLE
The form attribute shouldn't be assigned to the input when its value is undefined

### ISSUE THREAD

**Issue by rosenfeld** [thysultan/dio.js#68]
I'm closing #67 in favor of this more easily reproducible issue:

https://codesandbox.io/s/q360k5oj14

You can see that Dio.js behaves differently from React by toggling the imports.

**Comment by thysultan** [comment]
>I suspect React won't touch this attribute if its value is undefined and maybe Dio.js should do the same, what do you think?

Does this also apply to `null`?

**Comment by rosenfeld** [comment]
Yes, I just replaced undefined with "null" and "false" in the link above and the result was the same. Actually even if I pass a number, true or a string it doesn't change the output. I'm not sure what React really does here.

**Comment by rosenfeld** [comment]
It's more likely there's something deeply wrong with Dio.js here because it won't render anything, not just the radios, and it won't display any errors either, we just get a blank screen.

**Comment by thysultan** [comment]
Based what i found out `form` looks like a readonly property that accepts string attributes set via `setAttribute`. DIO trying to set it as a property `input.form = value` would probably throw an error. It probably doesn't change the output for React because of how the `form` property works(if there are not forms that match the string id noop)

>it won't display any errors either, we just get a blank screen.

Does this apply only to codesandbox? I get an error outside of codesandbox as well.

**Comment by rosenfeld** [comment]
I didn't get any errors locally

**Comment by rosenfeld** [comment]
Maybe you're trying against your recent changes to fix some errors when unmounting?

**Comment by rosenfeld** [comment]
By the way, I tried to add that line locally and it didn't seem to fix the error handling here...

**Comment by thysultan** [comment]
Yes, that must be it. I have a fix for this, just trying to find a reasonable test for it.

>By the way, I tried to add that line locally and it didn't seem to fix the error handling here...

Do you manage to reach the same point that i mentioned? Are we still talking about this issue or #69 ?

>TypeError: Cannot read property 'action' of undefined

**Comment by rosenfeld** [comment]
Yes, I tried following this:

"For example https://github.com/thysultan/dio.js/blob/master/src/Core/Commit.js#L151 putting a if (!element.active) return fixes the issue. "

**Comment by rosenfeld** [comment]
But I didn't try in that particular project but in my own.

**Comment by thysultan** [comment]
Can you update that issue with the error you got, assuming it's not the same error?

**Comment by rosenfeld** [comment]
Ok, let me try against the original project first and if it works there I'll try to change that project to reproduce the errors I'm getting in the other project.

**Comment by rosenfeld** [comment]
Yes, I can confirm that change fixed for that particular project, I'll now try to replicate the other errors I'm getting in that project.

**Comment by thysultan** [comment]
You can also try to point to the version in the [repo](https://github.com/thysultan/dio.js/blob/master/dist/umd.js) it has the latest as far as the patch i worked on on.

**Comment by thysultan** [comment]
Patched in 9.1.1
